src/test/scala/chiselTests/util/CatSpec.scala

```diff
@@ -68,7 +67,8 @@ class CatSpec extends ChiselFlatSpec {
       // noPrefix to avoid `out` as prefix
       out := Cat(in)
     }
-    val chirrtl = ChiselStage.emitChirrtl(new MyModule)
+    val chirrtl = ChiselStage.emitCHIRRTL(new MyModule)
+    // This test can fail when run from IntelliJ because the source locator path can be different
     chirrtl should include("cat(in[0], in[1]) @[src/test/scala/chiselTests/util/CatSpec.scala")
```

Comment:
Nit: stripping the source locator seems better.

Reply:
Will do in next commit